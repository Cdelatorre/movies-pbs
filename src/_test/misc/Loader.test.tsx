import { render, screen } from "@testing-library/react";
import Loader from "../../components/misc/loader";

describe("Renders Loader component correctly", () => {
  test("Renders Loader default component correctly", () => {
    render(<Loader />);

    const containerLoader = screen.getByTestId("loader");
    const loader = screen.getByText("Loading...");

    expect(containerLoader).toBeInTheDocument();
    expect(loader).toBeInTheDocument();

    expect(containerLoader).toMatchSnapshot();
  });
});
