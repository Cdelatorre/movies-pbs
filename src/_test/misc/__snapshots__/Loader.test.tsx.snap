// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Loader component correctly Renders Loader default component correctly 1`] = `
<div
  data-testid="loader"
  id="loader"
>
  <div
    class="loading"
  >
    Loading...
  </div>
</div>
`;
